Android/app/src/main/java/com/mehequanna/mmiw/StatViewPagerFragment.kt

```diff
@@ -0,0 +1,74 @@
+package com.mehequanna.mmiw
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import kotlinx.android.synthetic.main.fragment_stat_viewpager.*
+
+private const val NUM_PAGES = 4
+
+class StatViewPagerFragment : Fragment() {
```

Comment:
Do we need this class anymore?

Reply:
No, but in my next PR that I'm putting up soon I will so if it's okay, I'm going to leave it for now